annotation-file-utilities/tests/ClassListAnnotationParameter.goal

```diff
@@ -1,7 +1,7 @@
 package annotator.tests;
 
 public class ClassListAnnotationParameter {
-  @java.lang.TestAnnotation({java.lang.Object.class,java.lang.String.class})
+  @java.lang.TestAnnotation({java.lang.Object.class, java.lang.String.class})
```

Comment:
It makes pull requests easier to read (now and in the future) if you put formatting changes in a separate pull request.

Reply:
This is one of the intended formatting changes of this PR.

Reply:
Got it, thanks.